services/yellow/src/ambassador/YellowServiceClient.cs

```diff
@@ -23,7 +23,7 @@ public YellowServiceClient(IChannel channel)
         {
             var request = new AddPaymentMethodToOrderRequest(orderId, new PaymentMethod(creditCard));
             var (response, exception) =
-                await _channel.Request<Response<PaymentMethodToOrderAdded>>(request, cancellationToken);
+                await _channel.Request<Response<bool?>>(request, cancellationToken);
```

Comment:
Use `Response<bool>` and return `response`.

Reply:
Suggestion applied